test/cases/testdata/partialobjectdoc/test-partialobjectdoc-ref.yaml

```diff
@@ -0,0 +1,22 @@
+cases:
+- note: partialobjectdoc/ref
+  modules:
+  - |
+    package generated
+
+    p.q[k] = v {
+      k := ["foo", "bar"][v]
+    }
+    
+    # p.baz := 2
```

Comment:
To keep?

Reply:
Yep, I was doing some experimenting to generate different IRs; uncommenting 👍 .